src/web/modules/services/services.http.ts

```diff
@@ -11,17 +11,24 @@ import { formatErrorsForTemplate, ClientFormError } from '../common/validationEr
 import UpdateOrganisationFormRequest from './UpdateOrganisationForm'
 import { IOValidationError } from '../../../lib/errors'
 
+function filterRealLiveServices(services: Service[]) {
+  return services.filter(service => service.current_go_live_stage === 'LIVE'
+    && !service.internal
+    && !service.archived)
+}
+
 const overview = async function overview(req: Request, res: Response): Promise<void> {
-  const services: Service[] = await AdminUsers.services()
-  res.render('services/overview', { services })
+  const live = req.query.live === undefined ? true : req.query.live && req.query.live !== "false"
```

Comment:
Potentially clarifying the name may be useful too, query param can be concise but a more specific name would help reading the following lines
```suggestion
  const shouldFilterLiveServices = req.query.live !== "false"
```

Reply:
nice suggestions, I've made those changes